### ISSUE TITLE
RN 0.26 Compatibility

### ISSUE THREAD

**Issue by ajoshdee** [xinthink/react-native-material-kit#164]
HI , can you please update this, it such a waste if no one can use this because it doesn't work on RN 0.26

**Comment by cablegunmaster** [comment]
in the folder node_modules , create a folder : "reat-native-material-kit" and put the content in there.
after that follow the install instructions and it works.

**Comment by ajoshdee** [comment]
can't you just bump the version to npm so it's easy to install?

**Comment by Crash--** [comment]
@xinthink has not a lot of time at the moment. So be patient. You can point to this github repo for npm instead of just react-native-material-kit by doing : `npm install xinthink/react-native-material-kit` 

I've no problem with 0.26 and MKButton on iOS.

**Comment by cyrbon** [comment]
> Although not all the things work in 0.26 (buttons have red borders ) I experienced.

On Android, I have experienced this on 0.25 until I've rebuild (`react-native run android`) and restarted (`react-native start`). Haven't tested on 0.26 yet, so perhaps it was something else.

**Comment by andreyluiz** [comment]
@cablegunmaster, your approach worked for me. :+1:

**Comment by AzizAK** [comment]
@cablegunmaster how can i put the content there? can you explain what is the content ?

**Comment by cablegunmaster** [comment]
@AzizAK you download the zip folder of this github and put it in the react-native-material-kit folder, from the master branch.

**Comment by AzizAK** [comment]
@cablegunmaster thank you very much! now its working correctly.

**Comment by alexprice91** [comment]
Never been this "cutting edge" with a technology before. Usually there are more mature libraries since the technology has been around for a while.

Any idea of how we can add maturity to `react-native-material-kit`?

**Comment by Crash--** [comment]
@alexprice91 by submitting PR and by hoping that @xinthink can merge it or give collaborator access to some of us ;).

**Comment by alexprice91** [comment]
@Crash-- it looks like, in this example, master has already resolved the issue, just needs a release.

**Comment by cablegunmaster** [comment]
Gonna recheck the redborder issue, otherwise you are right @alexprice91 .

**Comment by alexprice91** [comment]
@cablegunmaster, thanks. And if you know how to fix the issue and can submit a PR, that sounds great. Otherwise I'll dig in and fix it.

**Comment by cablegunmaster** [comment]
@alexprice91  my JS skills not that high yet that I can fix it so far. -_-;; which I find very sad.

**Comment by alexprice91** [comment]
No problem. Is it (the redborder issue) still there?

**Comment by Crash--** [comment]
@alexprice91 It was just a generality about your question because the solution of this issue has been already given few times now ;).

**Comment by alexprice91** [comment]
Got it, I'm very tired this morning. I think this would be a smart move `give collaborator access to some of us`.

**Comment by xinthink** [comment]
Sorry for the delay, please use the master branch for this moment, as @Crash-- pointed out.
I'll upgrade to the latest RN version this weekend.
Apologize again!

And @alexprice91 @Crash-- is it ok if I grant you the collaborator access?

**Comment by alexprice91** [comment]
Yep, that sounds great!

**Comment by Crash--** [comment]
Yep! We'll be able to close Issues, PR and create tag if needed.

**Comment by Crash--** [comment]
To summarize the solution 
### First, remove installed version
`rm -rf ./node_modules/react-native-material-kit/`
### Then, install the master branch of the github repo (which containts the fixes)
`npm install  xinthink/react-native-material-kit` 
### Then link it to your project
`rnpm link`

**Comment by shankiesan** [comment]
I'm still having the red buttons and no-worky!

Anyone else still having issues after doing @Crash-- 's workaround?

**Comment by AzizAK** [comment]
@shankiesan my approach was 
1- uninstall the package 
2- install it and delete the node_modules all the files inside material kit folder
3-  download the material kit as zip from github
4- unzip and copy the files into the same file in node_modules

**Comment by Crash--** [comment]
@shankiesan I don't think "red buttons" is about RN 0.26 compatibility it's more about linking this lib in your project. rnpm should have done that for you automatically, can you check that? Do you use this lib before RN 0.26?

**Comment by shankiesan** [comment]
@AzizAK – I tried your approach and it didn't work, unfortunately

@Crash-- I assumed that `rnpm` was a typo when you meant to say `npm`. I've now discovered rnpm 😄 . I'll try this shortly. Cheers

**Comment by xinthink** [comment]
@alexprice91 I've invited you to join the [Slack channel](https://rnmk.slack.com) for this project. I think it's a better place to discuss the development of the project.
And @Crash-- would you mind send me your email address, so that I can invite you into the channel too?

**Comment by xinthink** [comment]
I tested with RN 0.26 & 0.27-rc, looks like the problem is solved, and I published v0.3.2 today.

I had the 'red-box' problem at first, but it seems disappeared after making a clean build of the whole project (including RN sources).

**Comment by shankiesan** [comment]
Thanks for all your help @xinthink @AzizAK and @Crash-- . Upgrading to 0.3.2 and then clearing all my caches under `~/.gradle/caches` did the track. Woop! 👍

**Comment by mdpedersen** [comment]
@xinthink, I'm having the "red-box" problem with a fresh install on react-native 0.29, I wonder if you can clarify your steps to a "clean build" which resolved the problem for you? Or is react-native 0.29 not supported?

**Comment by Crash--** [comment]
@mdpedersen RNMK is compatible with RN 0.29. 

Have you linked RNMK with `rnpm` ? If yes, have you this problem on iOS or Android?

**Comment by eladnava** [comment]
Strange, I'm getting a red box as well when using `MKSpinner`, using the example code provided.

**Comment by mdpedersen** [comment]
@Crash: Yes I've linked with rnpm, and I'm seeing the problem on Android (haven't tried iOS yet).

**Comment by eladnava** [comment]
In the meantime I went with [`ActivityIndicator`](https://facebook.github.io/react-native/docs/activityindicator.html)

**Comment by RichardLindhout** [comment]
It works fine on Android with reactie native a lot of the issues are because of linking with rpnm.

To fix this issue on iOS
After linking with rpnm run the code with xCode and after that it works.

To fix this issue on Android
After linking with rpnm change the package init etc to MainApplication instead of MainActivity!
